### PR TITLE
feat: delete resource

### PR DESCRIPTION
**Pull request by djaircarvalho**
gif-or-image

### What?

Delete resource

### Why?

So we can replicate the folder's deletion on keycloak

### CODE REVIEW

File: packages/keycloak/package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "@nestjs/graphql": "^7.6.0",
     "@nestjs/passport": "^7.1.0",
     "apollo-server-express": "^2.17.0",
+    "faker": "^5.1.0",
```
Comment (renatoaug) [inline packages/keycloak/package.json:28]: We already have it in `devDependencies` :thinking:

**Review by ndeitch — APPROVED** [review]

**Review by renatoaug — APPROVED** [review]

File: packages/keycloak/src/service/delete-resource.service.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import { DeleteResourceClient, GetClientToken } from '../client'
+
+@Injectable()
+export class DeleteResourceService {
+  constructor(
+    private readonly deleteResourceClient: DeleteResourceClient,
+    private readonly getClientToken: GetClientToken,
+  ) {}
+
+  async perform(realm: string, resourceId: string): Promise<void> {
```
Comment (renatoaug) [inline packages/keycloak/src/service/delete-resource.service.ts:11]: @djaircarvalho I believe that skore-one will send us the `name of the resource`, which is the `folder id`, and we will need to find the resource id through this endpoint:

```
http://${host}:${port}/auth/realms/${realm_name}/authz/protection/resource_set?name=folder_id
```

**Review by renatoaug — COMMENTED** [review]
@djaircarvalho I think

File: packages/keycloak/src/service/delete-resource.service.ts (hunk 2 above)
Comment (djaircarvalho) [reply packages/keycloak/src/service/delete-resource.service.ts:11]: good point

**Pull request merged**